### ISSUE TITLE
Define datasets structure

### ISSUE THREAD

**Issue by ondrejpudis** [opendatalabcz/traffic-surveys-automation#1]
## Definition

Finalize and document the structure of datasets - image resolution, frame rate and annotations.

We have three data sources:
- data from AI City Challenge (https://www.aicitychallenge.org/2021-data-and-evaluation)
    - contains labels which need to be properly transformed to match the defined standard
- recordings captured from live YouTube streams
    - manually annotated using https://github.com/openvinotoolkit/cvat tool 
- recordings provided by FD CVUT
    - numbers of passed cars provided with a part of the dataset

## Output

**Output is a code that generates a set of `tensorflow` datasets which generate data that can be fed into a model and used for training, fine-tuning and evaluation.**

## CVAT tool annotations

When annotated dataset is exported from CVAT it creates a `tfrecord` which contains the data with the following structure:
```
{
    "image/encoded": tf.io.FixedLenFeature([], tf.string),
    "image/filename": tf.io.FixedLenFeature([], tf.string),
    "image/format": tf.io.FixedLenFeature([], tf.string),
    "image/height": tf.io.FixedLenFeature([], tf.int64),
    "image/key/sha256": tf.io.FixedLenFeature([], tf.string),
    "image/object/bbox/xmax": tf.io.VarLenFeature(tf.float32),
    "image/object/bbox/xmin": tf.io.VarLenFeature(tf.float32),
    "image/object/bbox/ymax": tf.io.VarLenFeature(tf.float32),
    "image/object/bbox/ymin": tf.io.VarLenFeature(tf.float32),
    "image/object/class/label": tf.io.VarLenFeature(tf.int64),
    "image/object/class/text": tf.io.VarLenFeature(tf.string),
    "image/source_id": tf.io.FixedLenFeature([], tf.string),
    "image/width": tf.io.FixedLenFeature([], tf.int64),
}
```

Data from AI City Challenge need to be transformed to the similar structure.
